src/test/java/com/bandwidth/sqs/consumer/acknowledger/MessageAcknowledgerTest.java

```diff
@@ -79,13 +84,22 @@ public void testTransfer() {
     }
 
     @Test
+    public void testTimeout() {
+        MessageAcknowledger<String> messageAcknowledger =
+                new MessageAcknowledger(sqsQueueMock, RECEIPT_ID, Instant.now());
+        messageAcknowledger.getCompletable().blockingAwait();
+        assertThat(messageAcknowledger.getAckMode().blockingGet()).isEqualTo(AckMode.IGNORE);
+
```

Comment:
Unless the artist prefers using negative space to convey meaning, you could remove this whitespace.